index.html

```diff
@@ -20,28 +21,50 @@
         </style>
         <link rel="stylesheet" href="css/main.css">
 
+        <link rel="stylesheet" type="text/css" href="http://fonts.googleapis.com/css?family=Open+Sans:Light" />
+
     </head>
-    <body ng-app="dashboardApp">
+    <body ng-app="dashboardApp" ng-init="init()">
         <!--[if lt IE 7]>
             <p class="chromeframe">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> or <a href="http://www.google.com/chromeframe/?redirect=true">activate Google Chrome Frame</a> to improve your experience.</p>
         <![endif]-->
 
         <!-- This code is taken from http://twitter.github.com/bootstrap/examples/hero.html -->
 
-        <div class="navbar navbar-inverse navbar-fixed-top">
-            <div class="navbar-inner">
-                <div class="container">
-                    <ul class="nav">
-                        <li>
-                            <span class="brand">MozWebQA Dashboard</span>
-                        </li>
-                        <li id="nav-issues"><a href="#/issues">GitHub Issues</a></li>
-                    </ul>
-                </div>
+        <nav class="navbar navbar-inverse navbar-fixed-top" style="background-color:#bf3c2e">
+          <div class="container-fluid">
+            <div class="navbar-header">
+              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+                <span class="sr-only">Toggle navigation</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+              </button>
+              <a class="navbar-brand" href="#" style="color:white">MozWebQA Dashboard</a>
+            </div>
+          </div>
+        </nav>
+
+        <div class="container-fluid">
+          <div class="row">
+            <div class="col-sm-3 col-md-2 sidebar">
+              <ul class="nav nav-sidebar">
+                <li id="nav-issues"><a href="#/issues">Issues<span class="sr-only">(current)</span></a></li>
+                <li id="nav-pullrequests"><a href="#/pullrequests">Pull Requests</a></li>
+              </ul>
+              <ul class="nav nav-sidebar">
+                <li id="nav-jenkins"><a href="https://webqa-ci.mozilla.com/">Jenkins</a></li>
+              </ul>
+              <ul class="nav nav-sidebar">
+                <li><a href="https://quality.mozilla.org/teams/web-qa/">About Web QA</a></li>
+                <li><a href="https://wiki.mozilla.org/QA/Execution/Web_Testing">Wiki</a></li>
+                <li><a href="https://oneanddone.mozilla.org/tasks/available/?search=&team=6">Contribute</a></li>
```

Comment:
Ignore that, the GitHub highlighting was throwing me off.. this is fine in a query string 😄